tests/main/interfaces-daemon-notify/task.yaml

```diff
@@ -32,7 +32,7 @@ execute: |
         fi
         sleep 1
     done
-    [ "$denials_before" -eq "$denials_after" ]
+    test "$denials_before" != "$denials_after"
```

Comment:
This looks incorrect, the `-eq` should translate to `=` here - or am I missing something?

Reply:
nice catch, thanks for the review